W-22951633 Remove ASM mentions (#1084)
mulesoft/docs-gateway: mule-gateway/modules/ROOT/pages/policies-automated-applying.adoc

You can configure any provided policies for Mule 4 as an automated policy. +
See the xref:api-manager::policies-ootb-landing-page.adoc[list of available Provided Policies] to learn how to configure each one of them.
+

> When configuring an automated policy, at the bottom of the policy configuration page, there is a *Rule of Applications* sub-section. This dictates the set of runtimes that will be affected by the policy. Configure the following parameters:

Reviewer: ```suggestion
When configuring an automated policy, at the end of the policy configuration page, there is a *Rule of Applications* sub-section. This section dictates the set of runtimes that the policy affects. Configure the following parameters:
```
